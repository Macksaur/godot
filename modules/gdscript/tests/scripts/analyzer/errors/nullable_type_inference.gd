func test():
    var bar := foo()
    print(bar.begins_with("")) # Should error


func foo() -> String?:
    return null