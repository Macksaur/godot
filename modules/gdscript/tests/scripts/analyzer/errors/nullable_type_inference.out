GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable "bar" to call "begins_with()".
